pygama/analysis/peak_fitting.py

```diff
@@ -115,75 +114,45 @@ def neg_poisson_log_like(pars, func, hist, bins, **kwargs):
-    FIXME: bin expected mean mu estimated by f(bin_center)*bin_width. Should
+    Note: bin expected mean mu estimated by f(bin_center)*bin_width. Should
     TODO: add option to integrate function over bin
     """
     mu = func(pgh.get_bin_centers(bins), *pars, **kwargs) * pgh.get_bin_widths(bins)
     return np.sum(mu - hist*np.log(mu))
+<<<<<<< HEAD
 
-
-def gauss(x, *args):
+#Define a gaussian distribution and corresponding neg log likelihood
+def gauss(x, mu, sigma, A=1):
     """
     define a gaussian distribution, w/ args: mu, sigma, area (optional).
     TODO: return the corresponding neg log likelihood
     """
-    # check inputs
-    if len(args) == 2:
-        mu, sigma, A = args, 1
-    elif len(args) == 3:
-        mu, sigma, A = args
-    else:
-        print("Incorrect usage! You input: {}".format(args))
-        exit()
-
-    # calculate gaussian
-    norm = A / (sigma * np.sqrt(2 * np.pi))
-    return norm * np.exp(-(x - mu)**2 / (2. * sigma**2))
+    return A * (1. / sigma / np.sqrt(2 * np.pi)) * np.exp(-(x - mu)**2 / (2. * sigma**2))
 
 
-def radford_peak(x, *args):
+def radford_peak(x, mu, sigma, hstep, htail, tau, bg0, a=1):
     """
     David Radford's HPGe peak shape function
     """
-    if len(args) == 6:
-        mu, sig, hstep, htail, tau, bg0, a = args, 1
-    elif len(args) == 7:
-        mu, sig, hstep, htail, tau, bg0, a = args
-    else:
-        print("Incorrect usage!  You input: {}".format(args))
-        exit()
-
-    # make sure the fractional amplitude parameters are reasonable
-    if htail < 0 or htail > 1:
-        return np.zeros_like(x)
-    if hstep < 0 or hstep > 1:
-        return np.zeros_like(x)
-
-    bg_term = bg0 # + x*bg1
-    if np.any(bg_term < 0):
-        return np.zeros_like(x)
+    #make sure the fractional amplitude parameters stay reasonable...
+    if htail < 0 or htail > 1: return np.zeros_like(x)
+    if hstep < 0 or hstep > 1: return np.zeros_like(x)
 
-    # compute the step and the low energy tail
-    step = a * hstep * erfc((x - mu) / (sig * np.sqrt(2)))
-    le_tail = h * htail
-    le_tail *= erfc((x - mu) / (sig * np.sqrt(2)) + sig / (tau * np.sqrt(2)))
-    le_tail *= np.exp((x - mu) / tau)
-    le_tail /= (2 * tau * np.exp(-(sig / (np.sqrt(2) * tau))**2))
+    bg_term = bg0  #+ x*bg1
+    if np.any(bg_term < 0): return np.zeros_like(x)
+    step = a * hstep * erfc((x - mu) / (sigma * np.sqrt(2)))
+    le_tail = a * htail * erfc(
+        (x - mu) / (sigma * np.sqrt(2)) + sigma / (tau * np.sqrt(2))) * np.exp(
+            (x - mu) / tau) / (2 * tau * np.exp(-(sigma /
+                                                  (np.sqrt(2) * tau))**2))
+    return (1 - htail) * gauss(x, mu, sigma, a) + bg_term + step + le_tail
 
-    # add up all the peak shape components
-    return (1 - htail) * gauss(x, mu, sig, a) + bg_term + step + le_tail
 
-
-def xtalball(x, *args):
+# power-law tail plus gaussian https://en.wikipedia.org/wiki/Crystal_Ball_function
```

Comment:
Same comment here.  We don't want comments above functions like this, it's not good style